### PR TITLE
[tests] add test to #10500

### PR DESCRIPTION
**Pull request by EndangeredMassa**
Adding a test to https://github.com/vercel/vercel/pull/10500

### CODE REVIEW

**Review by trek — APPROVED** [review]

File: packages/cli/test/helpers/prepare.js [hunk 1]
```diff
@@ -415,6 +415,23 @@ module.exports = async function prepare(session, binaryPath, tmpFixturesDir) {
...
+        settings: {
+          framework: null,
+        },
+      }),
+      'package.json': JSON.stringify({
+        scripts: {
+          build: 'mkdir -p public && echo hi > public/index.txt',
+        },
+        dependencies: {
+          '@vercel/speed-insights': 'latest',
```
Comment (styfle) [inline packages/cli/test/helpers/prepare.js]: Does this get installed? If so, lets use a specific version here

**Review by cb1kenobi — APPROVED** [review]
LGTM

File: packages/cli/test/helpers/prepare.js (hunk 1 above)
Comment (EndangeredMassa) [reply packages/cli/test/helpers/prepare.js]: yeah, good call; updated

**Review by styfle — APPROVED** [review]

**Comment by EndangeredMassa** [comment]
huh, I had this test passing locally; will look into it

**Comment by tobiaslins** [comment]
@EndangeredMassa thanks!
The issue is that our package is private and you most probably have the NPM_TOKEN on your local machine. Lets just map it to some random library `"@vercel/speed-insights": "npm:server-only"` or something. This should be enough. 

We can change this later on, once we published the package!

**Comment by EndangeredMassa** [comment]
Yeah, makes sense. I just updated it to skip the install step for this test.

**Pull request merged**